material-intro-screen/src/main/java/agency/tango/materialintroscreen/MaterialIntroActivity.java

```diff
@@ -69,6 +71,7 @@
     private SparseArray<MessageButtonBehaviour> messageButtonBehaviours = new SparseArray<>();
 
     @Override
+    @CallSuper
```

Comment:
I checked in IntroActivity and without @CallSuper annotation, a developer isn't forced to call super.onCreate(), so I have added that in MaterialIntroActivity.